src/combat.ts

```diff
@@ -1,5 +1,5 @@
-import { mpCost, myClass, toInt } from "kolmafia";
-import { $item, $skill, get, have, StrictMacro } from "libram";
+import { mpCost, myClass, myPrimestat, toInt } from "kolmafia";
+import { $item, $skill, $stat, get, have, StrictMacro } from "libram";
 
 export const mainStat = myClass().primestat;
```

Comment:
Instead of using `myPrimestat()` everywhere (invoking a function call each time), why are we not using this exported const?

Reply:
I'm not seeing this updated, and I'm also unclear on what you're referring to in "required a stat output"

Reply:
Hmmm. Tried again.